include/map.h

```diff
@@ -133,8 +152,8 @@ typedef struct map {
 	goal_t goal;                                       /**< goal in the map*/
 	player_t player[MAP_MAX_NUM_OF_PLAYERS];           /**< player in the map*/
 	map_textures_t textures;                           /**< map element textures*/
-	player_t player1;                                   /**< player in the map*/
-	map_textures_multi_t textures1;                           /**< map element textures*/
+	player_t player1;                                  /**< player in the map*/
```

Comment:
We already include all the players, the same as what we did for barriers and arrows. 
player_t player[MAP_MAX_NUM_OF_PLAYERS];  
You don't need to declare a player and its structure separately.